recipes/lrez/build.sh

```diff
@@ -1,23 +1,29 @@
 #!/bin/bash
-export HTSLIB_ROOT=${PREFIX}/lib
-rm -rf build
-mkdir build
-cd build
-cmake ..
-make -j2
-cd ..
-
-mkdir -p $PREFIX/bin/
 
+export LDFLAGS="${LDFLAGS} -L${PREFIX}/lib"
+export LIBS="-lz -lm -lc -lboost_iostreams"
+export CPLUS_INCLUDE_PATH=${PREFIX}/include
 
-cp build/tools/Compare ${PREFIX}/bin/
-chmod +x $PREFIX/bin/Compare
+mkdir -p ${PREFIX}/bin
 
-cp build/tools/BamExtractor ${PREFIX}/bin/
-chmod +x $PREFIX/bin/BamExtractor
+# Install bamtools
+cd bamtools
+mkdir -p build
+cd build
+cmake -DBUILD_SHARED_LIBS=ON -DCMAKE_INSTALL_PREFIX=$PWD/../ ..
+make
+make install
+cd ../
+if [ -d "lib64" ]
+then
+    	mv lib64 lib
+fi
+cp ${SRC_DIR}/bamtools/lib/*.so* ${PREFIX}/lib
```

Comment:
```suggestion
cp "${SRC_DIR}/bamtools/lib/"*".${SHLIB_EXT}"* "${PREFIX}/lib/"
```
On macOS you'd get `.dylib` instead of `.so`. `conda-build` sets the [`SHLIB_EXT`](https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html#environment-variables-set-during-the-build-process) variable you can use here.
You might need this for `liblrez` below too